client/images/mitre-logo.svg

```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="utf-8"?>
 <!-- Generator: Adobe Illustrator 24.0.1, SVG Export Plug-In . SVG Version: 6.00 Build 0)  -->
 <svg version="1.1" id="Layer_1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px"
-	 viewBox="0 0 240 92" style="enable-background:new 0 0 240 92;" xml:space="preserve">
+	 viewBox="0 0 240 92" width="240" height="92" style="enable-background:new 0 0 240 92;" xml:space="preserve">
```

Comment:
When I download the PDF using Safari, the logo appears to be cut-off. This wasn't an issue on Chrome nor Microsoft Edge. Do you know why this might be?
<img width="605" alt="PDF_Download_Safari" src="https://user-images.githubusercontent.com/51176978/138325125-24e4fc75-af7b-4b59-9e8b-2e4396f6505f.png">

Reply:
test latest commit on safari and chrome, looks like PNG fixed the resizing error